tests/iob.test.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
-require('should');
+var should = require('should');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['should' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1243400235/issues/source?bid=2452379&fileBranchId=2921449#l3)